about.html

```diff
@@ -0,0 +1,128 @@
+                
+                <p class="mt-5 text-center">Please contact us per Email for any further questions about CC Global Summit 2022</p>
+                <a class="text-black fw-bold mb-5" href="mailto:travelworld@travel.com">travelworld@travel.com</a>
+            </div>            
+        </section>
+
+        <section class="container d-flex flex-column align-items-center mb-5">
+            <h3 class="mt-5 text-center">Creative Commons Global Summit 2022 Logo</h3>
+            <p class="text-center mt-4">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, mollitia optio placeat suscipit excepturi totam! <br> Temporibus vel maxime distinctio laboriosam harum dolore veritatis incidunt odit ut, doloremque atque illo eos.</p>
+            <a class="navbar-brand text-dark mx-5 border border-4 p-5" href="#"><i class="fas fa-tram fa-3x">travel</i></a>
+        </section>
+        <div class="vector-container d-flex justify-content-center mb-4">
+            <div class="vector-separate"></div>
+        </div>
+
+        <section class="d-flex flex-column align-items-center mb-5">
+            <h3 class="text-center">See the past CC Global Summits</h3>
+            <div class="vector-container d-flex justify-content-center mb-4">
+                <div class="vector"></div>
+            </div>
+            <p class="text-center">Take a look at the last two CC Global Summits which took place in Buenos Aires and Warsaw</p>
+            <div class="d-lg-flex gap-5 mt-4">
+                <div class="img-1 d-flex flex-column align-items-center justify-content-center meet">
+                    <h2 class="text-white">2020</h2>
+                    <p class="text-white">CC Global Summit 2020 in Buenos Aires</p>
+                </div>
+                <div class="img-2 d-flex flex-column align-items-center justify-content-center">
+                    <h2 class="text-white">2021</h2>
+                    <p class="text-white">CC Global Summit 2021 in Warsaw</p>
+                </div>
+            </div>
+
+        </section>
+
+    </main>
+    <footer class="gap-5 d-lg-flex justify-content-end bg-dark py-3">
+        <a class="navbar-brand text-secondary mx-5" href="#"><i class="fas fa-tram fa-3x">travel</i></a>
+        <div class="mt-3 mx-5">
+            <p class="fw-bold text-white text-center">2022 Creative Commons Korea. Some Rigths Reserved. <span></span> </p>
+            <p class="text-white text-center" >Lorem ipsum dolor sit amet consectetur adipisicing elit. In quis quos ipsam, maxime voluptatum.</p>
+        </div>
+    </footer>
```

Comment:
The footer design of the about page should look identical to the footer design of the home page **except** for the colors. Please make sure the padding and margin are the same.